### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by jlmakes**
Hey Stéphane,

I’ve recently taken an interest in ASTs, exploring `acorn`, `acorn-walk`, and `acorn-jsx`. Like many others, I was confused when I attempted to walk my first tree and encountered:

```
TypeError: baseVisitor[type] is not a function
```

You must be aware of this since you created `acorn-jsx-walk`. Yet, the author of `acorn-jsx` seems opposed to including built-in support for `acorn-walk`. I think the demand is clear for a packaged solution.

_I contacted you, because I would like to update `acorn-jsx-walk` with the recent community contributions that have failed to land in `acorn-jsx`. Since you seem open to this idea, I opened this pull request to begin sharing my progress._

### CODE REVIEW

**Comment by jlmakes** [comment]
This is a complete example showing how `acorn-jsx-walk@2` could work:

```js
// Create Acorn parser with JSX support
const acorn = require("acorn")
const jsx = require("acorn-jsx")
const parser = acorn.Parser.extend(jsx())

// Create Acorn walker with JSX support
const acornWalk = require("acorn-walk")
const jsxWalk = require("acorn-jsx-walk")
const walk = jsxWalk(acornWalk)

// Create AST from source containing JSX
const source = `
  const a = 2
  const fn = () => {
    const el = <div>Hello world!</div>
    return el
  }
`
const ast = parser.parse(source)

// Finally...
walk.simple(ast, {
  JSXElement(node) {
    console.log(`Found a ${node.type}!`)
  }
})

```

This is based off of @adrianheine’s proposal to `acorn-jsx`

**Comment by sderosiaux** [comment]
Hey @jlmakes !

Yes, I get it. For a smooth evolution, I'd like to make you admin of the project, is that okay with you?
Nowadays, I'm working quite far from the front-end world so I won't be effective.

**Comment by jlmakes** [comment]
Sure thing, Stéphane!

File: README.md [hunk 1]
```diff
@@ -39,14 +39,12 @@ walk.simple(ast, {
...
-Open source under the [MIT License](https://github.com/sderosiaux/acorn-jsx-walk/blob/master/LICENSE).
-
-Special thanks to @sderosiaux and @adrianheine.
+<p>
+Copyright 2019 <a href="https://github.com/sderosiaux/acorn-jsx-walk/graphs/contributors">Contributors</a>
+	<br>
+Open source under the <a href="https://github.com/sderosiaux/acorn-jsx-walk/blob/master/LICENSE">MIT License</a>.
+	<br>
+Special thanks to <a href="https://github.com/sderosiaux">@sderosiaux</a> and <a href="https://github.com/adrianheine">@adrianheine</a>.
+</p>
```
Comment (jlmakes) [inline README.md]: Woo, getting that formatting right was tough! 😅

**Comment by jlmakes** [comment]
Sounds good to me @sderosiaux!

I think I'm ready to begin testing the first pre-release. When you transfer admin rights to repo, can you also transfer ownership of `acorn-walk-jsx` on NPM ([~jlmakes](https://www.npmjs.com/~jlmakes))?

**Comment by jlmakes** [comment]
Sounds good to me @sderosiaux!

I think I'm ready to begin testing the first pre-release. When you transfer admin rights to repo, can you also add me to `acorn-walk-jsx` on NPM ([~jlmakes](https://www.npmjs.com/~jlmakes))?

**Comment by sderosiaux** [comment]
I just added you on npm, I hope it's okay ;-)

**Comment by jlmakes** [comment]
Almost ready to publish `2.0.0-canary.1` thanks for your support @sderosiaux 
Do you mind updating the repo description to match the new package description?

> JSX support for Acorn Walk

**Comment by sderosiaux** [comment]
Awesome work @jlmakes 👍

**Pull request merged**